life-writings/modernized/09-chapter-7.adoc

```diff
@@ -442,15 +482,19 @@ Seeing I could not be heard, it came upon me to write these lines following to t
 which were delivered into Edward Aiscough`'s hand,
 who said he would read them to the rest of his brethren.
 
-For the King`'s Justices, now sitting at Castor, these.
+[.embedded-content-document.letter]
+--
+
+[.blurb]
+=== For the King`'s Justices, now sitting at Castor, these.
 
 Friends,--In tender love I send you these lines, that you may consider,
 that God beholds you in the seat of judgment.
 Therefore be wise, and not rash and hasty in judgment, lest,
 by untrue surmises and false reports, you condemn the innocent.
 For I am one that fears the Lord, loves the king,
 and seeks the peace of the land of my nativity,
-and therefore ought to have the benefit of the wholesome laws to which I was horn,
+and therefore ought to have the benefit of the wholesome laws to which I was born,
```

Comment:
I just happened to see this and assumed it should be born, not horn, but i didn't check it in the original because I don't have it so I thought you could decide if that was right.